.pylintrc

```diff
@@ -8,4 +8,7 @@ min-similarity-lines=10
 argument-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
 attr-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
 method-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
-variable-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
\ No newline at end of file
+variable-rgx=_?_?(?:(?:[a-z0-9]+(?:_[a-z0-9]+)*_?_?)|(?:[a-z0-9]+(?:[A-Z][a-z0-9]*)*))$
+
+[DESIGN]
+max-positional-arguments=10
```

Comment:
Why did you add this argument here?
This was already added in `develop` branch, hence does it cause a conflict error?

Reply:
I think that commit is added as I merged the `kangz12345/318/update-pylintrc` branch into this branch. Other PRs that I lately merged also have that commits and there was no conflict!